packages/router/src/router.tsx

```diff
@@ -175,7 +175,8 @@ const LocationAwareRouter: React.FC<RouterProps> = ({
   )
 
   const shouldShowSplash =
-    (!hasHomeRoute && pathname === '/') || !hasGeneratedRoutes
+    ((!hasHomeRoute && pathname === '/') || !hasGeneratedRoutes) &&
+    process.env.NODE_ENV !== 'production'
```

Comment:
This is a sensical approach. Unfortunately, however, we cannot assume NODE_ENV will be set correctly in production (if set at all — e.g. Netlify does not automatically set NODE_ENV).

I think the logic needs to be something more like (pseudo code):
```
if (shouldShowSplash) {
    try {
         ...to show splash
    }
    catch { 
            ...if that fails revert to PageNotFound instead of "Something went wrong"
    }
}
```

You can test this locally in your project if you build and then run the api-server. E.g.:
```
$ yarn rw build // this builds for production
$ yarn rw serve // this starts a server for api and web
```

Reply:
You're right, I should not assume that.

>$ yarn rw build
$ yarn rw serve

I love the developer experience at Redwood.